src/ocean_csys.cpp

```diff
@@ -158,7 +158,7 @@ void oceancsys::ocean_csys_run( unitval tbox, unitval carbon )
     const double Tk = Tc + 273.15;
 
 	// Check that all is OK with input data
-	H_ASSERT( Tk > 265 && Tk < 308, "bad Tk value" ); // Kelvin
+	//H_ASSERT( Tk > 265 && Tk < 308, "bad Tk value" ); // Kelvin
```

Comment:
Why commenting this out?

Reply:
I think this was a left over from when I was debugging something.